src/transformState.js

```diff
@@ -61,8 +61,26 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
-function transformState(state, transforms) {
-  // write code here
+ function transformState(state, transforms) {
+  for (const key in transforms) {
+    if (transforms[key].operation === 'clear') {
+      for (const stateKey in state) {
+        delete state[stateKey];
+      }
+    }
```

Comment:
```suggestion
    if (transforms.operation === 'clear') {
      for (const property in state) {
        delete state[property];
      }
    }
```
Теперь  можно не использовать [key]. Можно property вместо stateKey.